src/styles.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star.active {
+  background: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
```

Remove active class number of active stars should enebled by `stars--n` modifier and `:nth-child`
Do not repeat your self too. 
```suggestion
.stars--n :nth-child {
  background: url("images/star-active.svg");
}
```